### DISCUSSION TITLE
In-place chat pop up only, without copying the code first.

### DISCUSSION THREAD

**Discussion by brownLlama** [CopilotC-Nvim/CopilotChat.nvim#35] [General]
Hey there, I have a question on in-place chat pop up. In the documentation, it is said, that we are supposed to copy some code in visual mode and then run the command `:CopilotChatInPlace` and then type my prompt, which is already cool and nice and my question is, if it is possible to only open the this chat pop up and just ask questions and go back and forth like in chatgpt for example. Thanks.

**Comment by brownLlama** [comment]
So, I only want like Prompt and copilot section. In one of your video, I learned how to toggle on/off Help section. So is there a way to toggle on/off Original section?
![CleanShot 2024-02-09 at 12 06 45](https://github.com/CopilotC-Nvim/CopilotChat.nvim/assets/81180451/a32bb363-0163-4ef3-af24-8696f535f5af)

**Reply by jellydn** [reply]
This `CopilotChatInPlace` command is designed for chatting with Copilot Chat and replace the original selection. I guess you are asking for a new chat handler, instead of `split`, it will be a popup handler.

cc @gptlang @ziontee113

**Reply by brownLlama** [reply]
I'm sorry but I didn't understand you fully. My question was, if I could have only those 2 sections in red boxes?? Thanks.

**Reply by jellydn** [reply]
> only those 2 sections in red boxes
Yes, that's possible.

I tried to say the current `CopilotChatInPlace` is NOT designed for that purpose. We should have `CopilotChatPopup` command as your suggestion.

**Reply by brownLlama** [reply]
@jellydn 
could you point out the `CopilotChatPopup` command in the documentation? I can't seem to figure out where it is or how to use it. Thanks again.

**Reply by jellydn** [reply]
We don't have this feature yet. I am raising this feature on https://discord.com/channels/1200633211236122665/1200633212041449606/1206149784717627462
